frontend/src/pages/modelServing/screens/projects/InferenceServiceModal/DataConnectionFolderPathField.tsx

```diff
@@ -30,25 +39,32 @@ const DataConnectionFolderPathField: React.FC<DataConnectionFolderPathFieldProps
     return () => clearTimeout(timer);
   }, [folderPath]);
   return (
-    <FormGroup
-      fieldId="folder-path"
-      label="Path"
-      isRequired
-      helperText={'Enter a path to a model or folder. This path cannot point to a root folder.'}
-      helperTextInvalid={'The path must not point to a root folder.'}
-      helperTextInvalidIcon={<ExclamationCircleIcon />}
-      validated={validated}
-    >
+    <FormGroup fieldId="folder-path" label="Path" isRequired>
       <InputGroup>
-        <InputGroupText variant="plain">/</InputGroupText>
-        <TextInput
-          aria-label="folder-path"
-          type="text"
-          value={folderPath}
-          placeholder="Example, data_folder"
-          onChange={handlePathChange}
-        />
+        <InputGroupText>/</InputGroupText>
```

Comment:
The variant `plain` is missing here

<img width="799" alt="Screenshot 2023-11-01 at 1 27 49 PM" src="https://github.com/opendatahub-io/odh-dashboard/assets/37624318/d1ace42b-167a-46d6-a8b1-0b2e722c7a68">